gradle/libs.versions.toml

```diff
@@ -38,6 +38,7 @@ wire = "5.0.0"
 bestPracticesPlugin = { id = "com.autonomousapps.plugin-best-practices-plugin", version = "0.10" }
 buildConfig = { id = "com.github.gmazzo.buildconfig", version = "5.3.5" }
 compose = { id = "org.jetbrains.compose", version.ref = "compose-jb" }
+compose-compiler = { id = "org.jetbrains.kotlin.plugin.compose", version.ref = "kotlin" }
```

Comment:
Looks like this doesn't get used. Can we remove it?

Reply:
Yes, I removed it.